app/helpers/ui_helper.rb

```diff
@@ -74,14 +74,14 @@ def print_button_to(link, options = {})
   end
 
   # Generic Submit button for a form
-  def submit_button(options = {})
```

Comment:
I'm a little concerned that this change will break other stuff

Reply:
The only thing that I changed is to use custom `name`. Usually it is set to `button`. So If name is not passed, I just set the `name` to `button`. So, I don't think this will break any other stuff.

Reply:
Also, the `data` parameter is used to send `confirm_message`.

Reply:
This is ok by me :+1: